bin/use_bazel_go.sh

```diff
@@ -1,9 +1,10 @@
 # This file should be sourced before using go commands
 # it ensures that bazel's version of go is used
 
-SP=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+ROOT=$( cd "$(dirname "${BASH_SOURCE[0]}")/.." ; pwd -P )
+DIR_NAME="$(basename ${ROOT})"
 
-TARGETBIN=$SP/../bazel-pilot
+TARGETBIN="${ROOT}/bazel-${DIR_NAME}"
```

Comment:
why  dir can be other names?

Reply:
Please use `bazel info bazel-bin`

Reply:
cloud builder checks out the code into /tmp/workspace and therefore the name is bazel-workspace

Reply:
Oh cool, didn't know `bazel info` did that. Yeah it's definitely better to use here: it gives you the full path to where the files that are symlinked into bazel-bin actually live on the file system. Should be more reliable than this.

Reply:
@kyessenov that s not bazel-bin that s bazel-pilot